### PR TITLE
Align bundles table according to the spec

### PR DESCRIPTION
**Pull request by barrymcgee**

### CODE REVIEW

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-gce.jujugui.org:8080//job/juju-gui/8410/

**Comment by barrymcgee** [comment]
## QA

- Run jujugui with 'profile' flag - `guiproxy -env prod --flags profile`
- Please compare bundles table on profile - `http://192.168.64.3:8042/u/$USERNAME/#bundles`
- Cross-ref with the design spec - https://github.com/CanonicalLtd/juju-design/blob/master/Profile/14-Bundles-table-specs.png

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-gce.jujugui.org:8080//job/juju-gui/8411/

File: jujugui/static/gui/src/app/components/profile/expanded-content/test-expanded-content.js [hunk 1]
```diff
@@ -79,7 +79,6 @@ describe('Profile expanded content', function() {
       <div className="profile-expanded-content">
         <div>Top row</div>
         <div className="six-col">
-          <p>logstash-core description</p>
```
Comment (hatched) [inline jujugui/static/gui/src/app/components/profile/expanded-content/test-expanded-content.js]: Can you add a test where the description is rendered plz.

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-gce.jujugui.org:8080//job/juju-gui/8412/

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-gce.jujugui.org:8080//job/juju-gui/8414/

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-gce.jujugui.org:8080//job/juju-gui/8415/

**Comment by hatched** [comment]
:shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci-gce.jujugui.org:8080/job/juju-gui-merge

**Pull request merged**